authentication_document.py

```diff
@@ -65,7 +66,7 @@ def __init__(self, _db, id, title, type, service_description, color_scheme,
         self.website = self.extract_link(
             rel="alternate", require_type="text/html"
         )
-        self.online_registration = self.extract_link(rel="register") is not None
+        self.online_registration = self.has_link(rel="register")
```

Comment:
This is a judgment call on my part; I figure if there is a single authentication mechanism that has a `register` link, we can treat the library as one that offers online registration.